SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -67,17 +56,15 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal
     If enabled, shuts down running deployments when you close the Unreal Editor.
 
     * **Snapshot path**</br>
-    Use this to specify the filepath of your Unreal GDK snapshot. If you leave this empty, it defaults to `<ProjectFileLocation>/../spatial/snapshots`.
+    Use this to specify the filepath of your snapshot. If you leave this empty, it defaults to `<GameRoot>/../spatial/snapshots`.
 
     * **Snapshot file name**</br>
-    Name of your SpatialOS Unreal GDK snapshot file.
+    Name of your snapshot file.
 
 * **Schema Generator**
 
     * **Generate Schema for all Supported Classes**</br>
-    **Experimental** Use this to generate schema for all `AActor` and `UActorComponent` classes (which *don't* extend `USceneComponent`) without explicitly tagging them as `SpatialType` within their UCLASS defintion. 
-    `USceneComponent` is unsupported due to there often being multiple of these components on an Actor which isn't currently supported.
-    If you have `UActorComponent`s which don't extend `USceneComponent` but exist in multiple on an Actor, *this can potentially crash your game*. Make sure there are never multiple components of the same type on an Actor. If there are make sure you modify your Actor to only have one, or use the explicit `SpatialType` tagging approach and don't tag the duplicate `UActorComponent`.
+    **Experimental** Use this to generate schema for all UObjects that have replicated or handover properties. As the GDK does not currently support multiple replicated Actor components of the same type on an Actor, no schemas will be generated for classes extended from `USceneComponent`.
```

Comment:
A couple more comments - sorry.

UObjects -> `UObjects`?

will be generated -> are generated (or, if possible, change it into the active: "X doesn't generate...")